README.md

```diff
@@ -240,9 +240,18 @@ pip3 install --pre torch torchvision --extra-index-url https://download.pytorch.
 
 and functorch
 ```
+pip3 install ninja  # Makes the build go faster
```

Comment:
Maybe we should stick to either `pip` or `pip3` everywhere.

Reply:
good point